src/components/Button/Button.stories.tsx

```diff
@@ -117,52 +120,45 @@ SecondarySmall.args = { variant: 'secondary', size: 'sm' };
 
 export const Tertiary = Template.bind({});
 Tertiary.args = {
-  children: 'Button',
```

Comment:
The `children` prop is defined in the `args` at the top of the file so I deleted this while I was in here.